Makefile

```diff
@@ -168,6 +169,29 @@ host-agent-binary: $(RELEASE_DIR)
 		-o ./bin/$(notdir $(RELEASE_BINARY))-$(GOOS)-$(GOARCH) $(HOST_AGENT_DIR)
 
 
+##@Release
+
+$(RELEASE_DIR):
+	rm -rf $(RELEASE_DIR)
+	mkdir -p $(RELEASE_DIR)
+
+build-release-artifacts: build-cluster-templates build-infra-yaml build-metadata-yaml build-host-agent-binary
+
+build-cluster-templates: $(RELEASE_DIR) cluster-templates
+	cp $(BYOH_TEMPLATES)/v1beta1/cluster-template.yaml $(RELEASE_DIR)/cluster-template.yaml
+	sed -i -e 1,7d $(RELEASE_DIR)/cluster-template.yaml
```

Comment:
I think this should be `1,20d`
https://github.com/vmware-tanzu/cluster-api-provider-bringyourownhost/blob/ebe8e3ea866260fa7b1312c4209a6a775a03d797/test/e2e/data/infrastructure-provider-byoh/v1beta1/cluster-template.yaml#L1-L20

Reply:
So I will assume `ClusterResourceSet` so not needed because it's again related to the CNI config which we want to clean up.
Ref: https://cluster-api.sigs.k8s.io/tasks/experimental-features/cluster-resource-set.html

Reply:
Yes, right. In the guide, we are asking to apply CNI separately at the end. So we can remove from cluster-template. atm, CRS is used only to apply CNI in the e2e test